src/Collections-Tests/TAddTest.trait.st

```diff
@@ -36,7 +37,7 @@ TAddTest >> test0FixtureRequirementsOfTAddTest [
 { #category : #'tests - adding' }
 TAddTest >> testTAdd [
 	| added collection |
-	collection :=self otherCollection .
+	collection :=self otherCollection.
```

Comment:
If you want to improve the formating maybe you can also add a space after the assignation opperator.

Reply:
I already did that on several places. Maybe I missed some - will do another pass.